js_modules/dagit/packages/core/src/instance/InstanceBackfills.tsx

```diff
@@ -30,7 +30,9 @@ const PAGE_SIZE = 10;
 export const InstanceBackfills = () => {
   useTrackPageView();
 
-  const queryData = useQuery(INSTANCE_HEALTH_FOR_BACKFILLS_QUERY);
+  const queryData = useQuery(INSTANCE_HEALTH_FOR_BACKFILLS_QUERY, {
+    fetchPolicy: 'cache-first', // Is this correct?
+  });
```

Comment:
This seems like it should be `cache-and-network`. @prha wdyt?

Reply:
I'm going to remove this, since it's querying for daemon health, and that should certainly be updated by the backend.